payment-request/PaymentAddress/attributes-and-toJSON-method-manual.https.html

```diff
@@ -0,0 +1,109 @@
+<!doctype html>
```

Comment:
Should this directory be renamed to ContactAddress?

Reply:
good catch!